ert3_examples/spe1/stages.yml

```diff
@@ -7,6 +7,9 @@
     -
       record: wells
       location: wells.json
+    -
+      record: spe1case2_data
+      location: SPE1CASE2.DATA.jinja2
```

Comment:
At this point we don't really need the information about which case it is anymore. It is not relevant. So, actually a better location with the now possibility to distinguish between names within the stage and data names I think something like this would be good:

```yml
-
  record: spe1_template
  location: SPE1.DATA.jinja2
```